### PR TITLE
Add documentation for CLI commands

### PR DESCRIPTION
**Pull request by alexandergraul**
Add documentation for all available CLI commands to the manpage.

Fix https://github.com/openSUSE/trollolo/issues/83.

### CODE REVIEW

File: man/trollolo.1.md [hunk 1]
```diff
@@ -36,6 +36,13 @@ and cards and has functionality for extracting data for burndown charts.
 
 ## COMMANDS
 
+### help -- Display help
+
+`trollolo help [COMMAND]`
+
+Display help and exit. Can take a command as an argument to display detailed
```
Comment (Ana06) [inline man/trollolo.1.md]: I don't think it is needed saying that the command exit :wink: and it should be display**s** . What aboud this:

_Displays help about the available trollolo commands. Can take a command as an argument to display detailed information about it._

File: man/trollolo.1.md [hunk 2]
```diff
@@ -83,16 +97,124 @@ conventions for Scrum boards, and reports burndown data.
...
+Create boards, lists and labels with names configured in '~/.trollolorc' or
+with the defaults.
+
+### move-backlog -- Move backlog from planning to sprint board
+
+`trollolo move-backlog --planning-board-id=<planning board id>
+--sprint-board-id=<sprint board id>`
+
+Move cards from the 'Backlog' list on the planning board to the 'Sprint
+Backlog' list on the sprint board.
```
Comment (Ana06) [inline man/trollolo.1.md]: What about mentioning here that this is what is doing at the beginning of the sprint once the planning meeting is finished? :thinking:

File: man/trollolo.1.md [hunk 3]
```diff
@@ -83,16 +97,124 @@ conventions for Scrum boards, and reports burndown data.
...
+
+### cleanup-sprint -- Move cards back to planning board
+
+`trollolo cleanup-sprint --board-id=<board id> --target-board-id=<target board
+id>`
+
+Move cards from 'Sprint Backlog', 'Doing' and 'QA' lists on the sprint board
+to the 'Ready' list on the planning board.
```
Comment (Ana06) [inline man/trollolo.1.md]: What about adding that this is what is done when the Sprint finished, after the review, as the unfinished card are moved to the backlog to work on them in future sprints?

File: man/trollolo.1.md (hunk 2 above)
Comment (alexandergraul) [reply man/trollolo.1.md]: I think anyone that uses this tool to manage their Scrum should know when to move the cards from the planning board to the sprint board / the other way around but I can add a sentence for each (`move-backlog` and `cleanup-sprint`)

Comment (Ana06) [reply man/trollolo.1.md]: yes, but it is difficult to read. If we add that it is used in this case it is much clear.

**Pull request merged**